saleor/graphql/product/mutations/products.py

```diff
@@ -271,6 +272,16 @@ class ProductInput(graphene.InputObjectType):
     seo = SeoInput(description='Search engine optimization fields.')
     weight = WeightScalar(
         description='Weight of the Product.', required=False)
+    sku = graphene.String(description=dedent("""Stock keeping unit. Note: this
+    field is required only used for products without variants."""))
+    quantity = graphene.Int(
+        description=dedent("""The total quantity of this variant availablefor
```

Comment:
Missing space between `availablefor`

Reply:
The total quantity of a product available for sale. Note: this field is only used if a product doesn't use variants.